forest/src/daemon.rs

```diff
@@ -2,27 +2,29 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use super::cli::{block_until_sigint, Config};
-use async_std::{channel::bounded, sync::RwLock, task};
 use auth::{generate_priv_key, JWT_IDENTIFIER};
 use chain::ChainStore;
 use chain_sync::ChainMuxer;
 use fil_types::verifier::FullVerifier;
 use forest_libp2p::{get_keypair, Libp2pService};
 use genesis::{import_chain, initialize_genesis};
-use libp2p::identity::{ed25519, Keypair};
-use log::{debug, info, trace};
 use message_pool::{MessagePool, MpoolConfig, MpoolRpcProvider};
 use paramfetch::{get_params_default, SectorSizeOpt};
-use rpassword::read_password;
 use rpc::{start_rpc, RpcState};
 use state_manager::StateManager;
-use std::io::prelude::*;
-use std::path::PathBuf;
-use std::sync::Arc;
 use utils::write_to_file;
 use wallet::ENCRYPTED_KEYSTORE_NAME;
 use wallet::{KeyStore, KeyStoreConfig};
 
+use async_std::{channel::bounded, sync::RwLock, task};
+use libp2p::identity::{ed25519, Keypair};
+use log::{debug, info, trace};
+use rpassword::read_password;
+
+use std::io::prelude::*;
+use std::path::PathBuf;
+use std::sync::Arc;
+
```

Comment:
Thank you for separating these out! I usually do them in the opposite order, though, but either way, it really helps vs mixing them in. I also do similar in Cargo.toml and use `#` to separate them.

Reply:
Yeah, I would really like it if this was part of our style guide